torchvision/io/video_reader.py

```diff
@@ -145,7 +187,18 @@ def seek(self, time_s: float, keyframes_only: bool = False) -> "VideoReader":
             frame with the exact timestamp if it exists or
             the first frame with timestamp larger than ``time_s``.
         """
-        self._c.seek(time_s, keyframes_only)
+        if self.backend in ["cuda", "video_reader"]:
+            self._c.seek(time_s, keyframes_only)
+        else:
+            # handle special case as pyav doesn't catch it
+            if time_s < 0:
+                time_s = 0
+            temp_str = self.container.streams.get(**self.pyav_stream)[0]
+            offset = int(round(time_s / temp_str.time_base))
+            if not keyframes_only:
+                warnings.warn("Accurate seek is not implemented for pyav backend")
```

Comment:
Not related to this PR, but do you have plans to add accurate seek in PyAV?

Reply:
Yeah, but in a roundabout way.
(I need to do it manually)